packages/spright-components/src/chat/conversation/styles.ts

```diff
@@ -17,6 +17,10 @@ export const styles = css`
         border: ${borderWidth} solid ${applicationBackgroundColor};
```

Comment:
Do you want to hide the border too? It's subtle but I suspect we'd rather not delineate the conversation from its parent at all.
<img width="796" height="637" alt="image" src="https://github.com/user-attachments/assets/37b32357-8713-4739-9bc4-ae12b9bf2141" />

Reply:
I could see an argument that this would mean it would be better to call this setting `appearance='frameless'`. But since this is a spright component I don't think we should get bogged down in naming, I'm ok with hide-background for now and we can change it if we get new designs with different variations.

Reply:
Is the other available value `default`?

Reply:
> I kind of like "overlay" as it conveys intent for usage.

Yea `overlay` works here since the usage is still visual oriented (wouldn't want an enum over coupled to use-cases like appearance="for-header" or something). Also wouldn't want `frameless` because that isn't really aligned to our other uses of frameless. I think `overlay` works okay for this new kind of visual state 👍

I'd also follow the newer attribute pattern we have been using where `default`  is `undefined` (I'd like to go back and fix our existing appearance attrs at some point).

```
"default": undefined,
"overlay": "overlay"
```